src/Common/src/System/IO/ReadOnlyMemoryStream.cs

```diff
@@ -0,0 +1,154 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.IO
+{
+    /// <summary>Provides a <see cref="Stream"/> for the contents of a <see cref="ReadOnlyMemory{byte}"/>.</summary>
+    internal sealed class ReadOnlyMemoryStream : Stream
```

Comment:
Should we publicly expose a stream that takes either ROMemory<T> or Memory<T> to the ctor?

Reply:
See https://github.com/dotnet/corefx/issues/22404